Py/a_main.py

```diff
@@ -45,15 +45,15 @@ class Vars :
         }
 
 ##
-def merge_all_chromosoms_snps() :
+def merge_all_chromosomes_snps() :
     """ """
 
     ##
     df = pd.DataFrame()
     for i in range(1 , 22 + 1) :
         print(i)
-        snps_fp = dyr.unpkd_mfi / f'ukb_mfi_chr{i}_v3.txt'
-        _df = pd.read_csv(snps_fp , sep = '\s+' , header = None)
+        snps_fp = dyr.unpkd_mfi / f'ukb_mfi_chr{i}_v3.txt'  # May want to consider https://docs.python.org/3/library/os.path.html#os.path.join
```

Comment:
Where is dyr defined, btw?  I looked up above and at the imports and maybe I'm blind, but I didn't see it anywhere.